### PR TITLE
README: add HTTP limit note.

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,5 +120,14 @@ make apply
 make destroy
 ```
 
+### Notes
+
+#### Limitations of HTTP headers
+
+Large group counts will not work with reverse proxies that limit HTTP headers to 4K. These counts will return an error similar to that returned for a cookie based approach. "Normal" HTTP header setups will support 1M header.
```
Comment (kans) [inline README.md]: | Large group counts will not work with reverse proxies that limit HTTP headers to 4K.

The Ingress Controller limits headers to around ~8 Kb by default.  `groups` refer to LDAP or github - something that you connect to dex.  I think these docs would make more sense with other auth docs somewhere.


| These counts will return an error similar to that returned for a cookie based approach. 

The error that end users will see is something like: `400: Request Header Or Cookie Too Large` after they log in to console.  Also, we no longer use cookies for auth.

| "Normal" HTTP header setups will support 1M header.
A "vanilla" Tectonic cluster will run into the Ingress Controller limit mentioned above.

File: README.md [hunk 2]
```diff
@@ -122,5 +120,14 @@ make apply
...
+
+To work with large-client-header-buffers: set `tectonic-custom-error` on the Ingress Configmap to `4 64k` to increase the limit in Ingress to EG 64k. See the nginx docs on the flag.
```
Comment (kans) [inline README.md]: The configmap in question is actually called `tectonic-custom-error` in `tectonic-system`.

**Comment by zbwright** [comment]
Rob agrees that this does not belong in the README.md. I'll take a look next week, see where it should live, and open a new PR. leaving this open as a reminder to me to do so.

**Comment by sym3tri** [comment]
@zbwright I agree, README is not the best place.

**Comment by cpanato** [comment]
@zbwright Can I close this PR?

**Comment by zbwright** [comment]
yes - replaced by https://github.com/coreos/tectonic-docs/pull/20

**Pull request closed without merging**